python/tests/test_util.py

```diff
@@ -23,13 +23,26 @@
 Tests for functions in util.py
 """
 import itertools
+import math
 import pickle
 import unittest
 
 import numpy as np
 
 import tests.tsutil as tsutil
 import tskit.util as util
+from tskit import MUTATION_UNKNOWN_TIME
```

Comment:
I can see the rationale for MUTATION_UNKNOWN_TIME in C, but it seems a bit clunky in Python. Maybe we could use ``UNKNOWN_TIME`` in Python? I guess in general, if we every need to specify another unknown time we'd use this value, so is there any avantage to TSK_MUTATION_UNKNOWN_TIME vs TSK_UNKNOWN_TIME in C either?

Reply:
Agreed, reads better as just unknown_time Changed in https://github.com/tskit-dev/tskit/pull/672/commits/202d5df081c31732a47e8fc344678e6b7d828a17